lib/generate-suggestions.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "fs-extra";
+import glob from "glob";
+import { parse } from "@babel/parser";
+import traverse from "@babel/traverse";
+
+import { fetchComponents } from "./http/fetchComponents";
+
+async function generateSuggestions() {
+  const components = await fetchComponents();
+  const results: {
+    [compApiId: string]: FindResults;
+  } = {};
+
+  for (const [compApiId, component] of Object.entries(components)) {
+    if (!results[compApiId]) {
+      results[compApiId] = [];
+    }
+    const result = await findTextInJSXFiles(".", component.text);
+    results[compApiId] = [...results[compApiId], ...result];
+  }
+  console.log(JSON.stringify(results, null, 2));
```

Comment:
nit: maybe a comment or simple function called `outputToConsole` that just wraps this line to ensure that it's clear as a reader that the `console.log` IS the behavior here

Reply:
<img width="256" alt="image" src="https://user-images.githubusercontent.com/15795866/230448633-1577a539-2a9a-4588-a62a-945dddb46c3c.png">